daffodil-core/src/main/java/org/apache/daffodil/api/validation/ValidatorNotRegisteredException.java

```diff
@@ -27,6 +27,6 @@ public class ValidatorNotRegisteredException extends Exception {
    * @param name the requested validator factory name
    */
   ValidatorNotRegisteredException(String name) {
-    super("No api.validation.ValidatorFactory is registered as " + name);
+    super("No Daffodil ValidatorFactory is registered with name '" + name + "'. Available validators: " + String.join(", ", Validators.list()));
```

Comment:
line too long

Reply:
Fixed. I usually rely on sbt-scalafmt to alert me of long lines, but forgot this was Java code. So I opened DAFFODIL-3036 to consider something similar for our java code.